src/compiler.rs

```diff
@@ -162,24 +165,26 @@ impl AmberCompiler {
             );
         }
 
-        let mut result = result.join("\n") + "\n";
+        let mut res = result.join("\n") + "\n";
 
-        if !self.cli_opts.disable_format {
-            if let Some(formatter) = BashFormatter::get_available() {
-                result = formatter.format(result);
-            }
+        let filters = self.cli_opts.disable_postprocessor.iter()
+            .map(|x| WildMatchPattern::new(x)).collect();
+
+        let postprocessors = PostProcessor::get_default(self.cli_opts.clone());
+        let postprocessors = PostProcessor::filter_default(postprocessors, filters);
```

Comment:
These two lines could essentially be:
```suggestion
        let postprocessors = PostProcessor::get_filtered(self.cli_opts.clone(), filters);
```